randovania/games/prime1/logic_database/Phazon Mines.txt

```diff
@@ -1302,7 +1321,7 @@ Extra - aabb: [122.126015, -44.231895, -102.18276, 201.19897, 105.61425, -30.731
       # Easy kills - Wavebuster (no X-Ray): https://youtu.be/3Y2n-boOGUw Super Missiles: https://youtu.be/aprVv7pePYc
       Combat (Beginner) or Normal Damage ≥ 100
   > Pickup (Main Power Bombs)
-      # Skipping the Security Drone trigger to take 400 less damage during item acquisition: https://youtu.be/LRmJn2BAYwc
+      # Skipping the Security Drone trigger to take 400 less damage during item acquisition: https://youtu.be/rDCTXaNmypI
```

Comment:
Why revert to Vertigo's incomplete video?

Reply:
Vertigo has a setup in the description whereas the other video does not. Although since Vertigo doesn't actually show boosting, you can arguably keep the old one, put both vids in, or add an addendum in the comment to use a PB and boost.

Reply:
Okay. Vertigo's video is fine